boilerplate/app/navigators/app-navigator.tsx

```diff
@@ -27,6 +27,7 @@ export type NavigatorParamList = {
   welcome: undefined
   demo: undefined
   demoList: undefined
+  // 🔥 Your screens go here
```

Comment:
_Love_ the fire emoji idea, that's brilliant

Reply:
Agreed~

Reply:
Thinking about it more, I wonder if we should use different emoji for different generators? "Look for the 🔥 " is less useful if we reuse it a lot. (This might be the plan, just wasn't sure.)

Reply:
I hadn't put all that much thought into it beyond establishing it as an easy way to direct the users eyes to the lines they need to change. I think there's definitely room to iterate on it in the future!